governance-exercises/Polling Exercise #2 Response.md

```diff
@@ -0,0 +1,40 @@
+#### My Interpretation of the MIPs and Plan
+
+[MIP101 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) does not direct a facilitator to draft a poll in this instance. It states that "If there’s clear evidence or significant suspicion that the identity of an AD has been revealed, a FacilitatorDAO **must immediately derecognize the AD**." then goes on to add "FacilitatorDAOs **must err on the side of caution** and act in case there is **any kind of real possibility that the privacy of an AD is compromised**.". 
+
+The wording could be improved, but in order to "err on the side of caution" an off-boarding by the Facilitator would unfortunately be the appropriate response here. This is consistent with [previous off-boardings](https://forum.makerdao.com/t/governance-scope-actions-pertaining-to-delegates-in-light-of-mip102c2-sp7-approval/20946) of ADs, which were not polled.
+
+Additionally, the requirement for the AD's identity to remain hidden would make it impossible to guarantee a fair poll fair. Voters cannot be presented with all the evidence without risking the reveal of Clive's identity.
+
+#### Message to Clive
+
+```
+Author: You
+Date: Monday 24th July 2023
+```
+
+Unfortunately, the Atlas is quite clear here. Facilitators are required to "err on the side of caution" if there is a real possibility that an Aligned Delegate's identity has been compromised. I don't enjoy this part of the job, but the MIPs are quite clear on the subject.
+
+#### Message to Diane
+
+```
+Author: You
+Date: Monday 24th July 2023
+```
+
+No poll will be needed. Would you be willing to publicly confirm your request for a whistleblower bounty and provide the address to send the bounty to?
```

Comment:
Good catch that they should confirm it publicly. Otherwise opens up a lot of risk later if they claim to never have received the funds.